src/main/java/com/ibm/watson/developer_cloud/http/HttpMediaType.java

```diff
@@ -26,118 +26,118 @@
   /**
    * Field APPLICATION_ATOM_XML. (value is "application/atom+xml")
    */
-  public final static String APPLICATION_ATOM_XML = "application/atom+xml";
```

Comment:
We use the variables in this class when building the requests and that's why they are `public final static`.
 * `static` because we don't need a class
 * `public` because we to access them from `WatsonService` or specific services
* `final` because they are constants.